tools/lsp/Cargo.toml

```diff
@@ -32,6 +32,7 @@ path = "wasm_main.rs"
 name = "slint_lsp_wasm"
 
 [features]
+slint = [] # remove when MSRV > 1.70
```

Comment:
Is this really > 1.70 or is 1.70 itself also sufficient? If the later, then perhaps it would be easier to rebase onto master.